collectors/hra/collector.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import requests
+import sqlalchemy
+import os
+from datetime import date, timedelta
+import time
+from .. import base
+from .record import Record
+logger = logging.getLogger(__name__)
+
+parse_datetime = base.helpers.parse_datetime
+
+url = 'https://stage.harp.org.uk/HARPApiExternal/api/ResearchSummaries'
+hra_user = os.environ.get('HRA_USERNAME')
+hra_pass = os.environ.get('HRA_PASSWORD')
+
+s = requests.Session()
+
+
+def make_request(url, from_, to, filter=None):
+    # Updates filter is optional-categorical value
+    # 1 - returns all studies for the given period
+    # 2 - returns only studies published in the period
+    # 3 - returns only studies modified in the period
+    result_str = url + '?datePublishedFrom=' + str(from_) + '&datePublishedTo=' + str(to)
```

Comment:
use python string formatting methods (`''.format()`), not `+`, for string formatting.